ui/src/react-admin/modules/content-page/components/blocks/BlockOverviewNewspaperTitles/hooks/useIeObjectsNewspaperTitles.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from '@tanstack/react-query';
+import { QUERY_KEYS } from '~shared/types';
+import { fetchWithLogoutJson } from '~shared/helpers/fetch-with-logout';
+import { getProxyUrl } from '~shared/helpers/get-proxy-url-from-admin-core-config';
+import { NewspaperTitle } from '~content-blocks/BlockOverviewNewspaperTitles/BlockOverviewNewspaperTitles.types';
+
+export const useGetIeObjectsNewspaperTitles = () => {
+	return useQuery(
+		[QUERY_KEYS.GET_IE_OBJECTS_NEWSPAPER_TITLES],
+		() => fetchWithLogoutJson<NewspaperTitle[]>(`${getProxyUrl()}/ie-objects/newspaper-titles`),
+		{
+			keepPreviousData: true,
+			enabled: true,
```

Comment:
Since this data will never change, we can set a cache time in the client of 1 hour
```suggestion
			enabled: true,
			stateTime: 3600000
```